pom.xml

```diff
@@ -125,5 +126,28 @@
         <module>optimize</module>
       </modules>
     </profile>
+
+    <profile>
+      <id>skip-sbom</id>
+      <activation>
+        <property>
+          <name>quickly</name>
+        </property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.cyclonedx</groupId>
+            <artifactId>cyclonedx-maven-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>default-makeAggregateBom</id>
+                <phase>none</phase>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
```

Comment:
🔧 Rather than an entire profile, you can skip cyclone with [`cyclonedx.skip`](https://cyclonedx.github.io/cyclonedx-maven-plugin/makeAggregateBom-mojo.html#skip) property. You can specify it in the same way as the other `quickly` properties.

https://github.com/camunda/camunda/blob/68b81fb41c2f1ff54805d3ff73c4001754f0f16e/parent/pom.xml#L235-L247


Something like
```xml
<cyclonedx.skip>${quickly}</cyclonedx.skip>
```